lib/the_little_thinkers_space/content.ex

```diff
@@ -44,6 +52,16 @@ defmodule TheLittleThinkersSpace.Content do
 
   def get_upload(id), do: Repo.get(Upload, id)
 
+  def store_upload(%Plug.Upload{filename: filename, path: path}, user_id) do
```

Comment:
I recommend using `upload` for the thing in your database, and something like `file` for the file on the harddrive.

Otherwise it can be difficult for a new reader to see at a glance what is being modified.

Reply:
makes sense :)